CrowdinSDK/Classes/CrowdinSDK/Localization/Extractor/LocalLocalizationExtractor.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 final class LocalLocalizationExtractor {
     enum Strings: String {
         case LocalPlurals
-        case LocalizableStringsdict
+        case LocalizableStringsdict = "Localizable.stringsdict"
```

Comment:
Why this? Was there a different bug?

Reply:
Not for native SDK, but yes, it fixes a potential issues with extracting local localization)